version/version_test.go

```diff
@@ -42,6 +42,21 @@ func TestVersionCheck(t *testing.T) {
 	}{{
 		name:          "greater version (patch)",
 		actualVersion: &testVersioner{version: "v1.17.2"},
+	}, {
+		name:          "greater version (patch), no v",
+		actualVersion: &testVersioner{version: "1.17.2"},
+	}, {
+		name:          "greater version (patch), pre-release",
+		actualVersion: &testVersioner{version: "1.17.2-kpn-065dce"},
+	}, {
+		name:            "greater version (patch), pre-release, envvar override",
+		actualVersion:   &testVersioner{version: "1.15.11-kpn-065dce"},
+		versionOverride: "1.15.11",
+		wantError:       true,
+	}, {
+		name:            "greater version (patch), pre-release, envvar override, -0 hack",
```

Comment:
whats the `-0` hack for?

Reply:
It's _smallest_ reasonable pre-release version

Reply:
ok - so we're comparing two prerelease versions

thanks